frontend/app_flowy/assets/translations/fr-FR.json

```
@@ -123,7 +136,7 @@
       "failedMsg": "Assurez-vous d'avoir terminé le processus de connexion dans votre navigateur."
     },
     "google": {
-      "title": "CONNEXION GOOGLE",
+      "title": "CONNEXION via GOOGLE",
```

I think it would be better to use a full caps text here
```suggestion
      "title": "CONNEXION VIA GOOGLE",
```